src/data/gedi_database.py

```diff
@@ -42,9 +42,13 @@ def gedi_sql_query(
                 "ignore",
                 message="__len__ for multi-part geometries is deprecated and will be removed in Shapely 2.0",
             )
-            conditions += [
+            queries = [
```

Comment:
Oh this is great, thanks.

Reply:
Well, it's perhaps the thin end of a wedge :) I tried to avoid building an ORM whilst giving us some more flexibility.